diff -Naur old_extract/freetype-2.14.3/builds/unix/config.sub new_workspace/freetype-2.14.3/builds/unix/config.sub
--- a/builds/unix/config.sub	2026-03-22 10:10:54.000000000 -0500
+++ b/builds/unix/config.sub	2026-06-23 15:16:16.573692347 -0500
@@ -2012,6 +2012,7 @@
 	| es* \
 	| fiwix* \
 	| freebsd* \
+	| ethereal* \
 	| fuchsia* \
 	| genix* \
 	| genode* \
diff -Naur old_extract/freetype-2.14.3/builds/unix/configure new_workspace/freetype-2.14.3/builds/unix/configure
--- a/builds/unix/configure	2026-03-22 10:10:53.000000000 -0500
+++ b/builds/unix/configure	2026-06-23 15:17:43.221519971 -0500
@@ -5477,6 +5477,10 @@
   lt_cv_deplibs_check_method=pass_all
   ;;
 
+ethereal*)
+      lt_cv_deplibs_check_method=pass_all
+      ;;
+
 hpux10.20* | hpux11*)
   lt_cv_file_magic_cmd=$FILECMD
   case $host_cpu in
@@ -11490,6 +11494,15 @@
   version_type=linux # correct to gnu/linux during the next big refactor
   need_lib_prefix=no
   need_version=no
+  library_names_spec='$libname$release$shared_ext$versuffix $libname$release$shared_ext$major $libname$shared_ext'
+  soname_spec='$libname$release$shared_ext$major'
+  shlibpath_var=LD_LIBRARY_PATH
+  ;;
+
+ethereal*)
+  version_type=linux # correct to gnu/linux during the next big refactor
+  need_lib_prefix=no
+  need_version=no
   library_names_spec='$libname$release$shared_ext$versuffix $libname$release$shared_ext$major $libname$shared_ext'
   soname_spec='$libname$release$shared_ext$major'
   shlibpath_var=LD_LIBRARY_PATH
